lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -37,26 +47,21 @@ class FeatureCard extends React.Component {
 
     this.requests = [];
     this.state = {
-      campaigns: [],
-      current_campaign: '',
-      signup_animation: ''
+      signupAnimation: '',
```

Comment:
This should be a boolean state (like `isSignedUp`) and then the animation can be handled in the render method.